src/components/typography/headingH3/HeadingH3.jsx

```diff
@@ -0,0 +1,4 @@
+// Reusable component for all H3, takes in text as a prop
+export const HeadingH3 = ({ text, className }) => {
+  return <h3 className={className}>{text}</h3>;
+};
```

Comment:
Interesting to see that you made reusable components for the text elements on the page. Inspired me to also try this in another project.